eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1190,6 +1209,45 @@ public function loadContentDrafts(User $user = null)
         return $versionInfoList;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function loadContentDraftList(?User $user = null, int $offset = 0, int $limit = -1): ContentDraftList
+    {
+        $list = new ContentDraftList();
+        if ($this->repository->hasAccess('content', 'versionread') === false) {
+            return $list;
```

Comment:
Please, make sure `$this->countContentDrafts($user)` returns 0 in the same conditions.